compiler/damlc/stable-packages/lib/DA/Daml/StablePackages.hs

```diff
@@ -161,6 +162,23 @@ ghcTuple = package version1_6 $ NM.singleton (emptyModule modName)
       [ mkWorkerDef modName unitTyCon tyVars [(mkIndexedField 1, TVar tyVar)]
       ]
 
+ghcTupleCheck :: Package
```

Comment:
I'm guessing you added this because `GHC.Tuple.Check` has to be imported by the stable packages that use tuples, but there shouldn't be any references to `GHC.Tuple.Check` after LFConversion, so this shouldn't be necessary. Besides, I don't really like having an empty package/module in our dependency graph.
Please see https://github.com/digital-asset/daml/pull/15479 - I tested `//compiler/damlc/tests:integration-v1dev` locally without any issues.

Reply:
another way you can tell this stable package isn't necessary is that the package ids of "dependent" stable packages didn't change at all